_template/alert-policies/example-alert-policy/static-alert.yml

```diff
@@ -44,20 +44,26 @@ expiration:
   # Open "Loss of Signal" violation if signal is lost (Default: false)
   openViolationOnExpiration: true | false
   # Time in seconds; Max value: 172800 (48hrs), null if closeViolationsOnExpiration and openViolationOnExpiration are both 'false'
-  expirationDuration:
+  expirationDuration: 60
```

Comment:
the default for this field is `null` when the defaults for the above 2 fields are met (both == `false`) 

since this is the "normal" state, does it make sense to setup a value in this template? what happens if the above values are both `false` and we pass `60` here? would that be a failed mutation?